support/elasticsearch2_search.erl

```diff
@@ -442,26 +465,37 @@ map_query({text, Text}, Context) ->
         <<"title*^2">>
     ],
     Fields = z_notifier:foldr(#elasticsearch_fields{query = Text}, DefaultFields, Context),
+    DefOpCfg = m_config:get_value(mod_elasticsearch2, default_operator, Context),
+    DefaultOperator = case z_convert:to_upper(z_convert:to_binary(DefOpCfg)) of
+        <<"AND">> = DefOp -> DefOp;
+        <<"OR">> = DefOp -> DefOp;
+        _ -> <<"AND">>
```

Comment:
Should it be AND or OR per default? The default used to be OR I guess, seeing line 451.

Reply:
As discussed, defaulting to 'OR' now.